softwerkskammer/lib/groupsAndMembers/groupsAndMembersService.js

```diff
@@ -59,6 +47,28 @@ module.exports = {
     });
   },
 
+  removeMember: function removeMember(nickname, callback) {
+    this.getMemberWithHisGroups(nickname, (err, member) => {
+      if (err || !member) { return callback(err); }
+
+      function unsubFunction(group, cb) {
+        this.unsubscribeMemberFromGroup(member, group.id, cb);
```

Comment:
Hast Du dafür einen Test? Ich glaube nämlich, dass dieses "this" nicht funktioniert, weil die unsubFunction ohne target aufgerufen wird...

Reply:
Adlerauge! - ich fix es

Reply:
done